homeassistant/components/aquostv/media_player.py

```diff
@@ -209,12 +172,12 @@ def turn_off(self):
     @_retry
     def volume_up(self):
         """Volume up the media player."""
-        self._remote.volume(int(self._volume * 60) + 2)
+        self._remote.volume(int(self.volume_level * 60) + 2)
```

Comment:
`_volume` was renamed here to `_attr_volume` in `__init__`
and from `_volume`  to `volume_level` here

Reply:
Oops, true. That must be `_attr_volume_level` of course. Well spotted. Thanks @ludeeus

Reply:
`self._volume` was returning `volume_level`. I just forgot something else. See latest commit.

Reply:
volume_level just returns `_attr_volume_level`, use `_attr_volume_level` here